content/docs/1.0.x/manage/debug/index.md

```diff
@@ -9,14 +9,20 @@ aliases:
 Modifying the log level preserves more detailed information in the logs.
 This can be useful when analyzing execution problems for your project.
 
-To do this:
+You can set the log level in either of two ways:
 
-* Adjust the value of the `LOG_LEVEL` environment variable
-  in the shipyard controller to `debug`.
-  Alternately, you can adjust the `loglevel` Helm value to `debug`.
+* Adjust the `loglevel` value to `debug`
```

Comment:
The right capitalization is important here
```suggestion
* Adjust the `logLevel` helm value to `debug`
```

Reply:
Done